### ISSUE TITLE
Add binary request method to MockHttpServletResponse

### ISSUE THREAD

**Issue by steinarb** [mockrunner/mockrunner#61]
It would be nice if MockHttpServletRequest had  a method to access the outputstream content as a byte[].

I had to access the stream and cast it to get to the binary content:
```
    private byte[] getBinaryContent(MockHttpServletResponse response) throws IOException {
        MockServletOutputStream outputstream = (MockServletOutputStream) response.getOutputStream();
        return outputstream.getBinaryContent();
    }
```

**Comment by steinarb** [comment]
I created a pull request for this issue but it didn't link up with the issue: https://github.com/mockrunner/mockrunner/pull/65

**Comment by steinarb** [comment]
Note: The fix for this issue was released with mockrunner 2.0.2, so this issue should be closed.

**Comment by cemartins** [comment]
Thanks
